src/webgl/p5.Geometry.js

```diff
@@ -1003,5 +1016,18 @@ p5.Geometry = class Geometry {
     }
     return this;
   }
+
+  /**
+   * When using textures each material contains all the face indices for that texture.
+   * this function collects all the faces mapped to different textures in a single array
+   * @method collectFaces
```

Comment:
Is this meant to be used internally only? If so we can add the `@private` tag so it doesn't show up in the reference for users.